lib/identity_config.rb

```diff
@@ -449,6 +450,24 @@ def self.build_store(config_map)
     config.add(:voip_allowed_phones, type: :json)
     config.add(:voip_block, type: :boolean)
 
+    config.add(:irs_attempt_api_auth_tokens, type: :comma_separated_string_list) do |list|
+      salt = SecureRandom.hex(32)
+      cost = config.written_env[:scrypt_cost]
+
+      list.map do |value|
+        scrypt_salt = cost + OpenSSL::Digest::SHA256.hexdigest(salt)
+        scrypted = SCrypt::Engine.hash_secret value, scrypt_salt, 32
+        digested_token = SCrypt::Password.new(scrypted).digest
+
+        {
+          cost: cost,
+          salt: salt,
+          digested_token: digested_token,
+          token: value,
```

Comment:
this keep the original unhashed value in memory, right? I see that we're using it in specs, but I feel like the point of this PR was to remove that?